lib/azure/resource_management/windows_credentials.rb

```diff
@@ -135,8 +135,16 @@ def target_name
         xplat_creds_cmd = Mixlib::ShellOut.new('cmdkey /list | findstr AzureXplatCli | findstr \--0- | findstr -v common')
         result = xplat_creds_cmd.run_command
         target_names = []
+
         if result.stdout.empty?
-          raise "Azure Credentials not found. Please run xplat's 'azure login' command"
+          xplat_creds_cmd = Mixlib::ShellOut.new('cmdkey /list | findstr AzureXplatCli | findstr -v common')

```

Comment:
can you add a debug message here about being unable to find a credential with --0- and falling back to looking for any credential?